pkgs/development/libraries/tbb/2021_8.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchurl
+, cmake
+}:
+
+let
+  common = import ./common.nix { inherit lib; };
+in
+stdenv.mkDerivation rec {
+  pname = "tbb";
+  version = "2021.8.0";
+
+  outputs = [ "out" "dev" ];
+
+  src = fetchFromGitHub {
+    owner = "oneapi-src";
+    repo = "oneTBB";
+    rev = "v${version}";
+    hash = "sha256-7MjUdPB1GsPt7ZkYj7DCisq20X8psljsVCjDpCSTYT4=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  patches = [
+    # Fix musl build; vendored from https://github.com/oneapi-src/oneTBB/pull/899
+    ./musl.patch
+
+    # Fix/suppress warnings on gcc12.1 from https://github.com/oneapi-src/oneTBB/pull/866
+    (fetchurl {
+      url = "https://patch-diff.githubusercontent.com/raw/oneapi-src/oneTBB/pull/866.patch";
+      sha256 = "sha256-zT+tRPnPVGUqKvob0RWjASRm4bPYYLtaaqTzA4EdU/8=";
+    })
```

Comment:
`fetchpatch` should be used when pulling generated patches as `github` might change comment or patch context from time to time. While at it use newer `hash = "sha256-zT+tRPnPVGUqKvob0RWjASRm4bPYYLtaaqTzA4EdU/8=";` instead of older `sha256 = ...`.